demos/tutorial/intro.php

```diff
@@ -171,24 +188,27 @@ protected function init(): void
     );
 });
 
-$wizard->addStep('Persistence', function ($page) {
+$wizard->addStep('Persistence', function ($page) use ($app) {
     $t = \atk4\ui\Text::addTo($page);
     $t->addParagraph(
         <<< 'EOF'
             Once your model is defined, it can be re-used later with any generic view:
             EOF
     );
 
-    Demo::addTo($page)->setCodeAndCall(function (View $owner) {
+    Demo::addTo($page)->setCodeAndCall(function (View $owner) use ($app) {
         session_start();
 
-        $model = new \atk4\ui\demo\DemoInvoice(new \atk4\data\Persistence\Array_($_SESSION['x'] ?? []));
+        $model = new \atk4\ui\demo\DemoInvoice(new \atk4\data\Persistence\Array_($_SESSION['x'] ?? []), ['dateFormat' => $app->ui_persistence->date_format]);
```

Comment:
I think it would be more clean without `use ($app)`, ie. `$owner->getApp()->...`

Reply:
you are right. old habit 😄